src/pages/Inventory.page.js

```diff
@@ -9,6 +9,23 @@ export class InventoryPage extends BaseSwagLabPage {
 
     get addItemToCartBtns() { return this.page.locator('[id^="add-to-cart"]'); }
 
+    get inventoryItemName() { return this.page.locator('.inventory_item_name '); }
+
+    get inventoryItemPrice() { return this.page.locator('.inventory_item_price'); }
+
+    async selectSortOption(value) {
+        await this.productSorting.selectOption(value);
+    }
+
+    async assertSortingNames() {
+        return this.inventoryItemName.allTextContents();
+    }
+
+    async assertSortingPrices() {
```

Comment:
the name of this method should be changed. We can name smth to assert when we check/verifying smth inside of the method. And for a method with assertions: it is not a good idea to keep it in Page Object